Yank/utils.py

```diff
@@ -489,7 +492,14 @@ def _resolve_path(d, path):
         The value contained in the node pointed by the path.
 
         """
-        return reduce(lambda d,k: d[k], path, d)
+        # TODO: Make sure this is working
+        # Converted from old py 2.X `reduce` function
+        function = lambda d,k: d[k]
+        it = iter(path)
+        accum_value = d
+        for x in it:
+            accum_value = function(accum_value, x)
+        return accum_value
```

Comment:
Will this function as originally intended?

`reduce` was removed in Py3, the code here is the what the Py2 docs call the "roughly equivalent" code